pkg/ccl/backupccl/restore_job.go

```diff
@@ -1974,6 +1976,40 @@ func clusterRestoreDuringUpgradeErr(
 		binaryVersion)
 }
 
+func (r *restoreResumer) maybeWriteDownloadJob(
```

Comment:
Usually I like functions in the file that calls them when they only have one caller. But I wonder if in this case we want to keep this in restore_online.

Reply:
Done.